File: ff4j-spring-boot-starter/src/main/java/org/ff4j/spring/boot/domain/GroupDescApiBean.java
package org.ff4j.spring.boot.domain;


import lombok.Getter;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/**
 * Created by Paul
 *
 * @author <a href="mailto:paul58914080@gmail.com">Paul Williams</a>
 */
public class GroupDescApiBean implements Serializable {

    private static final long serialVersionUID = -7339190302097692175L;

    @Getter
    private String groupName;

    @Getter
    private List<String> features = new ArrayList<>();

    public GroupDescApiBean(String groupName, List<String> names) {
        this.groupName = groupName;
        this.features = names;
    }
}

File: ff4j-spring-services/src/main/kotlin/org/ff4j/services/domain/GroupDescApiBean.kt
package org.ff4j.services.domain

import java.io.Serializable

/**
 * Created by Paul
 *
 * @author <a href="mailto:paul58914080@gmail.com">Paul Williams</a>
 */
class GroupDescApiBean : Serializable {


    companion object {
        private const val serialVersionUID = -7339190302097692175L
    }

    var groupName: String? = null
    var features: MutableList<String> = ArrayList()

    constructor() : super()

    constructor(groupName: String, names: MutableList<String>) {
        this.groupName = groupName
        this.features = names
    }
}